distribution/std-lib/Standard/src/Image/Codecs/Internal.enso

```diff
@@ -1,31 +1,48 @@
 from Standard.Base import all
+
 import Standard.Image.Codecs
 
 polyglot java import org.opencv.core.MatOfInt
 polyglot java import org.opencv.imgcodecs.Imgcodecs
 
 ## PRIVATE
+Codecs.Read_Alpha_Channel.to_integer = Imgcodecs.IMREAD_UNCHANGED
+
+## PRIVATE
+Codecs.Read_Grayscale.to_integer = Imgcodecs.IMREAD_GRAYSCALE
+
+## PRIVATE
+Codecs.Read_Gdal.to_integer = Imgcodecs.IMREAD_LOAD_GDAL
+
+## PRIVATE
+Codecs.Write_Jpeg_Quality.to_vector =
+    [Imgcodecs.IMWRITE_JPEG_QUALITY, this.value]
 
-   Convert read flag to OpenCV.
-read_flag_to_integer flag = case flag of
-    Codecs.Read_Alpha_Channel -> Imgcodecs.IMREAD_UNCHANGED
-    Codecs.Read_Grayscale -> Imgcodecs.IMREAD_GRAYSCALE
-    Codecs.Read_Gdal -> Imgcodecs.IMREAD_LOAD_GDAL
+## PRIVATE
+Codecs.Write_Jpeg_Progressive.to_vector =
+    [Imgcodecs.IMWRITE_JPEG_PROGRESSIVE, 1]
 
 ## PRIVATE
+Codecs.Write_Jpeg_Optimize.to_vector =
+    [Imgcodecs.IMWRITE_JPEG_OPTIMIZE, 1]
 
-   Convert write flag to OpenCV.
-write_flag_to_vector flag = case flag of
-    Codecs.Write_Jpeg_Quality value -> [Imgcodecs.IMWRITE_JPEG_QUALITY, value]
-    Codecs.Write_Jpeg_Progressive -> [Imgcodecs.IMWRITE_JPEG_PROGRESSIVE, 1]
-    Codecs.Write_Jpeg_Optimize -> [Imgcodecs.IMWRITE_JPEG_OPTIMIZE, 1]
-    Codecs.Write_Jpeg_Luma_Quality value -> [Imgcodecs.IMWRITE_JPEG_LUMA_QUALITY, value]
-    Codecs.Write_Jpeg_Chroma_Quality value -> [Imgcodecs.IMWRITE_JPEG_CHROMA_QUALITY, value]
-    Codecs.Write_Png_Compression value -> [Imgcodecs.IMWRITE_PNG_COMPRESSION, value]
-    Codecs.Write_Webp_Quality value -> [Imgcodecs.IMWRITE_WEBP_QUALITY, value]
+## PRIVATE
+Codecs.Write_Jpeg_Luma_Quality.to_vector =
+    [Imgcodecs.IMWRITE_JPEG_LUMA_QUALITY, this.value]
+
+## PRIVATE
+Codecs.Write_Jpeg_Chroma_Quality.to_vector =
+    [Imgcodecs.IMWRITE_JPEG_CHROMA_QUALITY, this.value]
+
+## PRIVATE
+Codecs.Write_Png_Compression.to_vector =
+    [Imgcodecs.IMWRITE_PNG_COMPRESSION, this.value]
+
+## PRIVATE
+Codecs.Write_Webp_Quality.to_vector =
+    [Imgcodecs.IMWRITE_WEBP_QUALITY, this.value]
```

Comment:
These should be done by calling `this.to_integer`. Otherwise you have duplicated values.

Reply:
Ok, refactored using `to_integer`